### DISCUSSION TITLE
where is the restore file? and how to restore my original setting??

### DISCUSSION THREAD

**Discussion by PowerfulShell** [builtbybel/ThisIsWin11#157] [General]
where is the restore file? and how to restore my original setting??

this is so frustrating......i trust this app, apple so called fixed, it stucked at "fixing taskbar monitor smh smh@#%@".......i saw the log said it back up, it created a restored point, so i close it, and restart program, click restore button above fix issue button, this is where shit goes wrong, it restore back to win11 default ...... WTF??? didnt it restroe to my original setting>>????? 

please tell me, how to restore back to my original setting, as it stated in the log file, i cannot find it anywhere in the program folder........

**Comment by billz270** [comment]
I want to restore back to my original settings too, and am not getting any help. Please tell me if you manage to find a solution
